invenio/base/scripts/demosite.py

```diff
@@ -97,10 +98,8 @@ def populate(packages=[], default_data=True, files=None,
     from werkzeug.utils import import_string
     map(import_string, packages)
 
-    from invenio.ext.sqlalchemy import db
     print(">>> Going to load demo records...")
-    db.session.execute("""TRUNCATE "schTASK" """)
-    db.session.commit()
+    truncate_table("schTASK")
```

Comment:
Uhm, I changed it because `db.session.execute('''TRUNCATE "schTASK"''')` raises an exception in MySQL because of the double quotes, but without the double quotes you'll get an exception in PostgreSQL ._.